repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -108,6 +108,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="set-rackers-groups" type="xs:string" use="optional">
```

Comment:
I don't name things well. It is known.

Yes, this attribute enables/disables the feature, and, in this case, forces the user to provide a value for x-pp-groups.

I can split this into enable/disable the feature and set a value/don't set a value. Easy enough. I wasn't sure we wanted to, though. Then we have three loosely-coupled attributes which determine the desired behavior.